Lib/fontTools/feaLib/variableScalar.py

```diff
@@ -109,3 +115,88 @@ def add_to_variation_store(self, store_builder):
         store_builder.setSupports(supports)
         index = store_builder.storeDeltas(deltas)
         return int(self.default), index
+
+    def __sub__(self, other):
+        if not isinstance(other, (VariableScalar, float, int)):
+            return NotImplemented
+        new = VariableScalar()
+        new.axes = self.axes
+        for loc in self.values.keys():
+            if isinstance(other, (float, int)):
+                new.add_value(dict(loc), self.value_at_location(loc) - other)
+            elif other.does_vary:
+                new.add_value(
+                    dict(loc),
+                    self.value_at_location(loc) - other.value_at_location(loc),
+                )
+            else:
+                new.add_value(
+                    dict(loc),
+                    self.value_at_location(loc) - list(other.values.values())[0],
+                )
+        return new
+
+    def __add__(self, other):
+        if not isinstance(other, (VariableScalar, float, int)):
+            return NotImplemented
+        new = VariableScalar()
+        new.axes = self.axes
```

Comment:
what if other.axes != self.axes? Is that valid? If so, please add a test for that.